options.go

```diff
@@ -234,6 +236,21 @@ func WithIAMAuthN() Option {
 	}
 }
 
+// InstanceConnectionNameResolver resolves  This allows an application to replace the default
+// DNSInstanceConnectionNameResolver with a custom implementation.
+type InstanceConnectionNameResolver interface {
+	Lookup(ctx context.Context, name string) (instanceName instance.ConnName, err error)
```

Comment:
Resolver -> Resolve would be more Go-like.

Reply:
Yes. See review comment.